nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3939,8 +3976,7 @@ to the cluster. It provides an additional layer of security. This value is blank
 |`nifi.cluster.flow.election.max.candidates`|Specifies the number of Nodes required in the cluster to cause early election of Flows. This allows the Nodes in the cluster to avoid having to wait a
 long time before starting processing if we reach at least this number of nodes in the cluster.
 |`nifi.cluster.load.balance.port`|Specifies the port to listen on for incoming connections for load balancing data across the cluster. The default value is `6342`.
```

Comment:
This wasn't part of your PR, but I noticed that the default value of 6432 for nifi.cluster.load.balance.port is not technically correct.  There's maven filtering occurring in nifi-framework/nifi-resources/pom.xml that set the property during build-time to 7430. The resulting nifi.properties in nifi-assembly/target/nifi-1.8.0-SNAPSHOT-bin/nifi-1.8.0-SNAPSHOT/conf has:
```properties
nifi.cluster.load.balance.port=7430
```
The following code (from nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java) that reads this property will use the default of 6432 if the property is missing from nifi.properties:
```java
public InetSocketAddress getClusterLoadBalanceAddress() {
    try {
        String address = getProperty(LOAD_BALANCE_ADDRESS);
        if (StringUtils.isBlank(address)) {
            address = getProperty(CLUSTER_NODE_ADDRESS);
        }
        if (StringUtils.isBlank(address)) {
            address = "localhost";
        }

        final int port = getIntegerProperty(LOAD_BALANCE_PORT, DEFAULT_LOAD_BALANCE_PORT);
        return InetSocketAddress.createUnresolved(address, port);
    } catch (final Exception e) {
        throw new RuntimeException("Invalid load balance address/port due to: " + e, e);
    }
}
```

Reply:
@jtstorck oops. I did mean for that to be 6432 in all places. I started with 6432, then when coding things up I remembered the value incorrectly, using 7430. I realized this and thought that I had changed all of them to 6432 but I must have missed a spot.

Reply:
Just noting that the default value is 6342 not 6432.